integration/js/utils/WebdriverSauceLabs.js

```diff
@@ -72,6 +71,37 @@ const getSafariCapabilities = capabilities => {
   return cap
 };
 
+const getChromeOptions = capabilities => {
+  let chromeOptions = {};
+  if (capabilities.useFakeMedia) {
+    chromeOptions['args'] = ['--use-fake-device-for-media-stream', '--use-fake-ui-for-media-stream'];
+    if (capabilities.video) {
+      chromeOptions['args'].push('--use-file-for-fake-video-capture=' + fetchMediaPath(capabilities.video, capabilities.browserName));
```

Comment:
[For understanding]: Is this required for all tests, I think we did not have it earlier, hence asking for more understanding?